### PR TITLE
Add skin layout support for `HitErrorMeter`s

### PR DESCRIPTION
**Pull request by peppy**
Decided to not support migration for now. Getting config into the skin implementations to make it somehow work seems like it's going to be too much effort vs payoff for now. I think most people will be happy to use the new layout editor to perfect their setup.

Also decided to remove the icon realignment logic as it's hard to follow in its current implementation. May add this back in a better way going forward (we probably want some kind of `Drawable` class which automatically orientates itself to an upright position regardless of parent hierarchy).

Also note that this doesn't display so great in the component toolbox, but neither do several other components, so I think I'll fix that up in a follow-up PR (the toolbox basically needs a full `DrawableRuleset` with hits going on, *hopefully* sourced from the player screen itself).

### CODE REVIEW

**Review by bdach — COMMENTED** [review]
Seems mostly uncontroversial.

File: osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs [hunk 1]
```diff
@@ -63,33 +58,42 @@ private void load(OsuColour colours)
...
+                        Anchor = Anchor.CentreLeft,
+                        Origin = Anchor.CentreLeft,
                         Children = new Drawable[]
                         {
                             colourBarsEarly = new Container
                             {
-                                Anchor = Anchor.y1 | alignment,
-                                Origin = alignment,
+                                Anchor = Anchor.CentreLeft,
+                                Origin = Anchor.x2,
```
[the remark this answers is not in the document]
Comment (peppy) [reply osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs:88]: Ah yes, this was part of incremental changes I made to the old code because understanding what the values should be was quite hard. I missed updating this one.

File: osu.Game/Skinning/DefaultSkin.cs [hunk 2]
```diff
@@ -78,6 +79,23 @@ public override Drawable GetDrawableComponent(ISkinComponent component)
...
+                                        hitError.Anchor = Anchor.CentreLeft;
+                                        hitError.Origin = Anchor.CentreLeft;
+                                    }
+
+                                    var hitError2 = container.OfType<HitErrorMeter>().LastOrDefault();
+
+                                    if (hitError2 != null)
+                                    {
+                                        hitError2.Anchor = Anchor.CentreRight;
+                                        hitError2.Origin = Anchor.CentreLeft;
```
[the remark this answers is not in the document]
Comment (peppy) [reply osu.Game/Skinning/DefaultSkin.cs:96]: That's correct. Will leave a comment I guess.

**Review by bdach — APPROVED** [review]

**Comment by Bryceed** [comment]
12d89b1b-2c7b-45a4-a1f4-685f2ea5514e
This option don't makes sense no more, if the game doesn't respect the player's choice to display it

**Suggestion:** Change this checkbox option to a dropdown with "_Always display_", "_Don't display on legacy skins_ (default)" and "_Never display_" options.

\_\_\_\_\_
_P.S.: Sorry if I commented in the wrong place, I thought it was more correct to comment on the change than just open an issue to comment about it._
